### DISCUSSION TITLE
Using Selenium for Authenticated pages Lighthouse report - net :ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by ganeshjoe** [GoogleChrome/lighthouse#13654] [Q&A]
Hello all-

I'm using selenium for browser automation for E2E user journey and integrated with Lighthouse CLI to run LH report for each pages.

Though the solution works locally fine, while running it thru Gitlab (kubernetes runner) - displays below error

"Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct url and the server is properly responding to all requests (Details: net:: ERR_CERT_AUTHORITY_INVALID)" - Please note this is happening for https sites that Im trying to test in pipeline runs on alpine linux image.

please suggest how to resolve this issue or any leads on the same is much appreciated

**Comment by adamraine** [comment]
We don't officially support selenium (https://github.com/GoogleChrome/lighthouse/issues/1833) but it's neat that you got it working locally.

> Details: net:: ERR_CERT_AUTHORITY_INVALID

This looks like a genuine certificate authority error that's out of Lighthouse's control.

Unfortunately I can't be of much help here, but if I had to guess your CA is probably not set up correctly in whatever container/environment Gitlab is testing in.

**Comment by ganeshjoe** [comment]
I have a authentication session/cookie url passing from selenium to lighthouse command. But when we pass this url to lighthouse report command I'm getting this error . I have all valid certs but its still throwing error

**Reply by aakash3771** [reply]
could you please share code snippet of how you did it. I am trying to do the same
